src/components/FuturesMode/FuturesMode.css

```
@@ -0,0 +1,1140 @@
+/* FuturesMode.css
+ * Layout: mobile-first tabs; desktop (≥1024px) uses 3-column terminal grid.
+ */
+
+@keyframes slideInUp {
```

Stylelint (17.7.0)

[error] Expected keyframe name "slideInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)